packages/block-editor/src/components/publish-date-time-picker/index.js

```diff
@@ -27,6 +28,7 @@ function PublishDateTimePicker(
 				onClose={ onClose }
 			/>
 			<DateTimePicker
+				startOfWeek={ getSettings().l10n.startOfWeek }
```

Comment:
This is probably a bit nit-picky, but is `l10n` always truthy? I wasn't sure if this needed some optional chaining and passing in a fallback, something like: `getSettings()?.l10n?.startOfWeek ?? 0`?

From the looks of it, it _should_ always be available, but since we're depending on the added inline script, I was wondering if it's worth hardening slightly?

Reply:
So if the inline script isn't added at all then we'll still have this data because of the default settings object that `@wordpress/date` contains:

https://github.com/WordPress/gutenberg/blob/cc6ed9837cbdd1a0283e19fe9dd0f13de8b66e45/packages/date/src/index.js#L63

Technically someone could call `setSettings` with an object that is missing some attributes. I'd say it should be up to `@wordpress/date` to guard against that, though, since the contract it gives callers (via its [type definitions](https://github.com/WordPress/gutenberg/blob/trunk/packages/date/src/index.js#L49)) is that `l10n` exists.

Reply:
Awesome, thanks for confirming!